vnext/src/Libraries/Pressability/Pressability.windows.js

```diff
@@ -639,29 +649,39 @@ export default class Pressability {
     // [Windows
     const keyboardEventHandlers = {
       onKeyUp: (event: KeyEvent): void => {
+        const {onKeyUp} = this._config;
+        onKeyUp && onKeyUp(event);
+
         if (
           event.nativeEvent.code === 'Space' ||
           event.nativeEvent.code === 'Enter' ||
           event.nativeEvent.code === 'GamepadA'
         ) {
-          const {onPressOut, onPress} = this._config;
-
-          // $FlowFixMe: PressEvents don't mesh with keyboarding APIs. Keep legacy behavior of passing KeyEvents instead
-          onPressOut && onPressOut(event);
-          // $FlowFixMe: PressEvents don't mesh with keyboarding APIs. Keep legacy behavior of passing KeyEvents instead
-          onPress && onPress(event);
+          // flowlint sketchy-null:warn, sketchy-null-bool:off
```

Comment:
Why do these warnings normally show up?

Reply:
I have seen that defaltPrevented == undefined, unless preventDefault() was called. Flow complains about the null check.
Only disabling it for booleans.

Reply:
I think it would be better to add the explicitly separate checks instead of disabling the warning.

Apart from catching errors, I think a motivation for the rule is that it makes intent clear to the reader. Having checks for both makes it really clear we want to check against both boolean and the undefined state, instead of thinking it is always a boolean.